doc/man3/RSA_generate_key.pod

```diff
@@ -19,13 +21,17 @@ Deprecated:
 
 =head1 DESCRIPTION
 
-RSA_generate_key_ex() generates a key pair and stores it in the B<RSA>
-structure provided in B<rsa>. The pseudo-random number generator must
-be seeded prior to calling RSA_generate_key_ex().
+RSA_generate_multi_prime_key() generates a key pair and stores it in the
```

Comment:
"a key pair" is kind of ambiguous now. Previously one could interpret it "as pair of prime factors" or "public-private key pair". Now it's probably appropriate to be more explicit with "public-private key pair".

Reply:
Every 'new' comment is addressed except this. TBH, I don't fell this is ambiguous at all, since the phrase 'key pair' here must be 'an RSA public-private key pair'...

Reply:
As already said once, it's not exactly appropriate to expect that documentation reader is intimately familiar with terminology to recognize *implied* meanings. But whatever...